docs/book/src/cronjob-tutorial/testdata/emptycontroller.go

```diff
@@ -41,7 +41,7 @@ objects, so these are added out of the box.
 
 // CronJobReconciler reconciles a CronJob object
 type CronJobReconciler struct {
-	client.Client
+	client client.Client
```

Comment:
If you run :

https://github.com/kubernetes-sigs/kubebuilder/blob/32e0fdc918b9a21485d7b042833402a62d0480f4/Makefile#L82-L85

It will be lost.

We need to the change in the hack/docs so that we can easily keep the samples from the docs updated.

Could you please check it out?